explorer/src/lib/components/block-details/BlockDetails.css

```diff
@@ -0,0 +1,66 @@
+.block-details {
+  display: flex;
+  flex-direction: column;
+  gap: 1.25rem;
+}
+
+.block-details__header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.block-details__header-heading {
+  font-size: 1.5rem;
+  font-weight: 400;
+}
+
+.block-details__list {
+  display: inline-grid;
+  grid-template-columns: 1fr 3fr;
+  font-size: 0.875rem;
+  column-gap: 1.875rem;
+}
+
+.block-details__list-timestamp {
+  text-transform: uppercase;
+}
+
+.block-details__list-anchor {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0.0625rem;
```

Comment:
It's okay to have this, although the difference with or without the margin is neglectable and I would personally remove it.